https://issues.redhat.com/browse/ACM-22753 (#8181)
open-cluster-management/rhacm-docs: release_notes/acm_deprecate_remove.adoc

|===
|Product or category | Affected item | Version | Recommended action | More details and links

| {global-hub}

> | You no longer need to migrate the resources, `ConfigMaps` and `Secrets`, when you migrate managed clusters. 

Author: Was there a reason we remove the configmaps and secrets from the migrating managed clusters doc? 

Ex: did users previously have to manually migrate the configmaps and secrets and now it's done automatically, so we don't have to document it? 

Basically, I'd like to give a bit more of a "why we remove it from the doc" here in this doc removal table :)

> | You no longer need to migrate the resources, `ConfigMaps` and `Secrets`, when you migrate managed clusters. 

Reviewer: ```suggestion
| You are not required to migrate the resources, `ConfigMaps` and `Secrets` when you migrate managed clusters. 
```
